module6/freethrows/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: page
+title: Free Throws
+---
+
+During Intermission, you worked on earning a certification in fundamental C#.  This is one way to keep up with your technical skills during a job hunt.  Another way to stay current with your skills is by practicing 'Free Throws' - activities that will help you cement your existing skills into muscle memory.
```

Comment:
I might edit this from "existing skills" to "C# fundamentals". Because I think we want the free throws to be on really Mod 1 concepts, not connecting to a DB or MVC or other more advanced topics they are working on in their projects.